keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ClusterActiveServerGroup.kt

```diff
@@ -19,7 +19,7 @@ data class ClusterActiveServerGroup(
 data class LaunchConfig(
   val ramdiskId: String?,
   val ebsOptimized: Boolean,
-  val imageId: String,
+  val imageId: String?,
```

Comment:
imageId becomes nullable to indicate that the cluster handler expects metadata supplying the image.

as we discussed in person, this is almost certainly the wrong thing where the spec ends up always being out of sync with the current state of the world, and the upsert becomes responsible for no-oping when the metadata supplied image is correct